script/Arcueid/Day 10/10_00_ARC10_1.txt

```diff
@@ -843,7 +843,7 @@ Today's a weekday, after all.
 [sha:11a6995ae96fda14669fff2db4504da292d64ebe]{
 -- Page 61, Offset 24413.
 -- ドアを少しだけ開けて、廊下に誰もいない事を確認してから移動する。
-I crack the door, make sure there's nobody hanging around the hallway, and make my move.
+I creak the door open, to make sure there's nobody around in the hallway, then make my move.
```

Comment:
'Cracking a door' is an expression, I do not believe that 'creaking' a door is. Original seems fine to me.